core/src/main/scala/com/yahoo/maha/core/BaseUTCTimeProvider.scala

```diff
@@ -52,15 +52,18 @@ class BaseUTCTimeProvider extends UTCTimeProvider with Logging {
     var utcDayFilter = localTimeDayFilter
     var utcHourFilter = localTimeHourFilter
     var utcMinuteFilter = localTimeMinuteFilter
-    if (isDebugEnabled) info(s"Timezone: $timezone")
-    if (!validateFilters(localTimeDayFilter, localTimeHourFilter, localTimeMinuteFilter) || !timezone.isDefined) {
+    info(s"Timezone: $timezone")
+    val validFilters = validateFilters(localTimeDayFilter, localTimeHourFilter, localTimeMinuteFilter)
+    if (!validFilters || !timezone.isDefined) {
+      info(s"validateFilters is  $validFilters timezone is defined ${timezone.isDefined}")
```

Comment:
Maybe merge line 55 down into this log, since we only care about this issue on specific requests rather than all of them.

Reply:
changed.